app/jobs/fs/download_sent_message_related_messages_job.rb

```diff
@@ -16,6 +16,10 @@ def perform(outbox_message, from: nil, to: nil, fs_client: FsEnvironment.fs_clie
       0.step do |k|
         received_messages = fs_api.fetch_received_messages(sent_message_id: outbox_message.metadata['fs_message_id'], page: k + 1, count: batch_size, from: from, to: to)
 
+        if received_messages['messages'].none?
```

Comment:
Neviem uplne odhadnut dopady tohto ale rozumiem spravne, ze to sa ide do tej istej schranky len druhym sposobom ak nic nenajdeme? Kedze tieto related messages sa volaju dost casto tak to moze asi zdvojnasobit pocet requestov na FS nie?

Reply:
Áno, tak.

Čo sa týka počtu requestov, 2x nie je reálny odhad.
- Druhý request sa zavolá pri správe, kde sa nenašlo nič pri prvom volaní. To sa udeje iba pri maximálne niekoľkých správach denne - kým ešte naozaj na FS nemajú potvrdenku - pričom tento job spúšťame s 3 minútovým odstupom po odoslaní podania, takže väčšina tam vtedy už je - t.j. pri väčšine sa request navyše nespraví nikdy.
- Potom druhý prípad, kedy sa pravidelne zavolá aj druhý request, je v týchto veľmi zriedkavých prípadoch, keď správu nevidno v prvom requeste.

Teraz tento job spúšťame na všetky podania za posledných 7 dní (cca 1700) 2x do dňa. Denne táto úprava pridá rádovo max 10 requestov naviac.